neuralprophet/TorchProphet.py

```diff
@@ -0,0 +1,553 @@
+        range of days around the date to be included as holidays.
+        lower_window=-2 will include 2 days prior to the date as holidays. Also
+        optionally can have a column prior_scale specifying the prior scale for
+        that holiday.
+    seasonality_mode: 'additive' (default) or 'multiplicative'.
+    seasonality_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    holidays_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    changepoint_prior_scale: Not supported for regularisation in NeuralProphet,
+        please use the `_reg` args instead.
+    mcmc_samples: Not required for NeuralProphet
+    interval_width: Float, width of the uncertainty intervals provided
+        for the forecast. If mcmc_samples=0, this will be only the uncertainty
+        in the trend using the MAP estimate of the extrapolated generative
+        model. If mcmc.samples>0, this will be integrated over all model
+        parameters, which will include uncertainty in seasonality.
+    uncertainty_samples: Not required for NeuralProphet.
+    stan_backend: Not supported by NeuralProphet.
+    """
+
+    def __init__(
+        self,
+        growth="linear",
+        changepoints=None,
+        n_changepoints=25,
+        changepoint_range=0.8,
+        yearly_seasonality="auto",
+        weekly_seasonality="auto",
+        daily_seasonality="auto",
+        holidays=None,
+        seasonality_mode="additive",
+        seasonality_prior_scale=None,
+        holidays_prior_scale=None,
+        changepoint_prior_scale=None,
+        mcmc_samples=None,
+        interval_width=0.80,
+        uncertainty_samples=None,
+        stan_backend=None,
+        **kwargs,
+    ):
+        # Check for unsupported features
+        if seasonality_prior_scale or holidays_prior_scale or changepoint_prior_scale:
+            log.info(
+                "Using `_prior_scale` is unsupported for regularisation in NeuralProphet, please use the `_reg` args instead."
+            )
+        if mcmc_samples or uncertainty_samples:
+            log.info(
+                "Providing the number of samples for Bayesian inference or Uncertainty estimation is not required in NeuralProphet."
+            )
+        if stan_backend:
+            log.info("A stan_backend is not used in NeuralProphet. Please remove the parameter")
+        # Run the NeuralProphet function
+        super(TorchProphet, self).__init__(
+            growth=growth,
+            changepoints=changepoints,
+            n_changepoints=n_changepoints,
+            changepoints_range=changepoint_range,
+            yearly_seasonality=yearly_seasonality,
+            weekly_seasonality=weekly_seasonality,
+            daily_seasonality=daily_seasonality,
+            seasonality_mode=seasonality_mode,
+            # TODO: activate once PR merged prediction_interval=interval_width,
```

Comment:
still relevant?

Reply:
Yes, looks like. There is still no `prediction_interval` in forecaster.py on main.

Reply:
I re-added the workaround with converting the interval_width to quantiles if no quantiles are provided.